tests/test_output.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+from pathlib import Path
+
+import src.output as op
+
+
+def test_write_audio():
+    """
+    Tests write_audio function, takes in an audio tensor with a file path and writes the audio to a file.
+    """
+    import torch
+
+    test_path = "tests/data/test_audio.wav"
+    audio_path = "tests/data/test_audio.pt"
+    audio_list = torch.load(audio_path)
+
+    op.write_audio(audio_list, test_path)
+
+    assert Path(test_path).is_file() is True
```

Comment:
The `is True` is redundant. Just type `assert Path(test_path).is_file()`. This format already assumes the variable is a Boolean.